From 1fbb46db545fad72daab79d769f19a4ae4cea6e3 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Tue, 28 Feb 2023 20:24:42 +0100
Subject: [PATCH 02/N] makepkg: bash v4 compat

I think this is from a time when we only had bash v4 which didn't support "-f"
and can be dropped
---
 scripts/makepkg.sh.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/makepkg.sh.in b/scripts/makepkg.sh.in
index fca9b2c..5549f07 100644
--- a/scripts/makepkg.sh.in
+++ b/scripts/makepkg.sh.in
@@ -440,7 +440,7 @@ run_function() {
 
 		$pkgfunc &>"$logpipe"
 
-		wait -f $teepid
+		wait $teepid
 		rm "$logpipe"
 	else
 		"$pkgfunc"
